<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1b1d23;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="2,0">
       <item row="1" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QWidget" name="viewWidget" native="true">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(85, 170, 255);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QFrame" name="frameRayCast">
         <property name="styleSheet">
          <string notr="true">border-radius: 15px;
background-color: rgb(39, 44, 54);
border: none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QSlider" name="ambientSlider">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="AmbientLabel">
              <property name="font">
               <font>
                <family>Oswald Light</family>
                <pointsize>12</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Ambient</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_20">
            <item>
             <widget class="QSlider" name="diffuseSlider">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="DiffuseLabel">
              <property name="font">
               <font>
                <family>Oswald Light</family>
                <pointsize>12</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Diffuse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_19">
            <item>
             <widget class="QSlider" name="specularSlider">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="SpecularLabel">
              <property name="font">
               <font>
                <family>Oswald Light</family>
                <pointsize>12</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Specular</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="frame">
         <property name="styleSheet">
          <string notr="true">border-radius: 15px;
background-color: rgb(39, 44, 54);
border: none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="loadButton">
              <property name="font">
               <font>
                <family>-apple-system</family>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
  appearance: button;
  backface-visibility: hidden;
  background-color: #405cf5;
  border-radius: 6px;
  border-width: 0;
  box-shadow: rgba(50, 50, 93, .1) 0 0 0 1px inset,rgba(50, 50, 93, .1) 0 2px 5px 0,rgba(0, 0, 0, .07) 0 1px 1px 0;
  box-sizing: border-box;
  color: #fff;
  cursor: pointer;
  font-family: -apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Ubuntu,sans-serif;
  font-size: 100%;
  height: 35px;
  line-height: 1.15;

  outline: none;
  overflow: hidden;
  padding: 0 5px;
  position: relative;
  text-align: center;
  text-transform: none;
  transform: translateZ(0);
  transition: all .2s,box-shadow .08s ease-in;
  touch-action: manipulation;
  width:50%;
}
QPushButton:hover{
background-color: #283999;
}
QPushButton:pressed{
background-color: #1c2973;
}
</string>
              </property>
              <property name="text">
               <string>Load DICOM files</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/upload-square-svgrepo-com.png</normaloff>icons/upload-square-svgrepo-com.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QSlider" name="iso_slider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="isoValue">
                <property name="font">
                 <font>
                  <family>Oswald Light</family>
                  <pointsize>14</pointsize>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>ISO Value:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QRadioButton" name="surfaceRadio">
                <property name="font">
                 <font>
                  <family>Oswald SemiBold</family>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Surface Rendering</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rayCastRadio">
                <property name="font">
                 <font>
                  <family>Oswald SemiBold</family>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>RayCast Rendering</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="realTimeCheck">
                <property name="font">
                 <font>
                  <family>Oswald SemiBold</family>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Real Time</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="renderButton">
                <property name="font">
                 <font>
                  <family>-apple-system</family>
                  <pointsize>12</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
  appearance: button;
  backface-visibility: hidden;
  background-color: #405cf5;
  border-radius: 6px;
  border-width: 0;
  box-shadow: rgba(50, 50, 93, .1) 0 0 0 1px inset,rgba(50, 50, 93, .1) 0 2px 5px 0,rgba(0, 0, 0, .07) 0 1px 1px 0;
  box-sizing: border-box;
  color: #fff;
  cursor: pointer;
  font-family: -apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Ubuntu,sans-serif;
  font-size: 100%;
  height: 35px;
  line-height: 1.15;

  outline: none;
  overflow: hidden;
  padding: 0 5px;
  position: relative;
  text-align: center;
  text-transform: none;
  transform: translateZ(0);
  transition: all .2s,box-shadow .08s ease-in;
  touch-action: manipulation;
  width:100%;
}
QPushButton:hover{
background-color: #283999;
}
QPushButton:pressed{
background-color: #1c2973;
}
</string>
                </property>
                <property name="text">
                 <string>Render</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
